.travis.yml

```diff
@@ -75,30 +75,30 @@ matrix:
            # Run this with pytest because our `setup.py` runner does not
            # currently pick up doctests in docs/
            env: MAIN_CMD="py.test"
-                SETUP_CMD="--remote-data -k-figure --cov ./"
+                SETUP_CMD="--remote-data -k-figure --cov --cov-config sunpy/tests/coveragerc ./"
 
          - python: 3.6
            stage: Comprehensive tests
            env: SETUP_CMD='build_docs -w'
 
+         # Python 2.7 (implicit astropy 2)
+         - python: 2.7
+           stage: Comprehensive tests
+           env: SETUP_CMD='test'
+
          # Figure test runs use an env stored in the repo so don't have conda
          # requirements
          # Python 2.7 figure tests
          - python: 2.7
            stage: Comprehensive tests
            env: SETUP_CMD='test --figure --coverage' CONDA_DEPENDENCIES=''
 
-         # Python 2.7 non-figure tests
-         - python: 2.7
-           stage: Comprehensive tests
-           env: SETUP_CMD='test'
-
          # Python 3.5 figure tests
          - python: 3.5
            stage: Comprehensive tests
            env: SETUP_CMD='test --figure --coverage' CONDA_DEPENDENCIES=''
 
-         # Python 3.5 non-figure tests
+         # Python 3.5 with astropy 2
          - python: 3.5
            stage: Comprehensive tests
            env: ASTROPY_VERSION='lts'
```

Comment:
Does LTS work without `astorpy-ci-extras`. I forgot if we moved it to the sunpy channel.

Reply:
it should be able to pull it from deafults

Reply:
OK I CAN REMOVE IT FROM MORE FILES